static/src/javascripts/polyfill.io

```diff
@@ -0,0 +1 @@
+https://polyfill.guim.co.uk/v2/polyfill.min.js?features=es6,es7,default-3.6&flags=gated&callback=guardianPolyfilled
```

Comment:
Interested to know why you're serving this off a new hostname and not `https://assets.guim.co.uk/` with your other assets? This will incur connection and full roundtrip latency penalty.